content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -11,104 +11,241 @@ keywords:
+| <file>/upload</file> | **No** | Push | Media files pushed from the local dataset to <file>/upload</file> are then uploaded to Google Photos and not sorted into an album. Because <file>/upload</file> is a temporary storage location, it can not accurately synchronize from one task to the next. Pushing to this folder does not preserve metadata and can result in duplicated files, poor performance, file name instability. |
+{{< /truetable >}}
+
+### Selecting the Dataset and Organizing Files
+
+Next, select TrueNAS SCALE local dataset or create a new one to use as the source or destination.
+
+{{< expand "Creating a Dataset" "v" >}}
+{{< include file="/static/includes/CreateDatasetSCALE.md" >}}
+{{< /expand >}}
+
+Configure the local dataset file structure (for push tasks) or Google Photos library albums (for pull tasks) as required by your direction, mode, and target selections (see above).
 
 ## Creating the API Credentials
-On the [Google API dashboard](https://console.cloud.google.com/apis/dashboard), click the dropdown menu next to the Google Cloud logo and select your project.
+
+On the [Google API dashboard](https://console.cloud.google.com/apis/dashboard), click the dropdown menu to the right of the Google Cloud logo and select your project.
+
 If you do not have a project, click **NEW PROJECT** and enter a value in **Project name**, **Organization**, and **Location**.
+Click **Create**.
 
-![GooglePhotosAPINewProject](/images/SCALE/DataProtection/GooglePhotosAPINewProject.png "New Project")
+{{< trueimage src="/images/SCALE/DataProtection/GooglePhotosAPINewProject.png" alt="New Project" id="New Project" >}}
 
 ### Enable API
+
 After you select your project, click **Enabled APIs & Services** on the left menu, then click **+ ENABLE APIS AND SERVICES**.
 
-![GooglePhotosAPIEnableAPIsandServices](/images/SCALE/DataProtection/GooglePhotosAPIEnableAPIsandServices.png "Enable APIs and Services")
+{{< trueimage src="/images/SCALE/DataProtection/GooglePhotosAPIEnableAPIsandServices.png" alt="Enable APIs and Services" id="Enable APIs and Services" >}}
 
 Enter **photos library api** in the search bar, then select **Photos Library API** and click **ENABLE**.
 
-![GooglePhotosAPIPhotosLibraryAPISearch](/images/SCALE/DataProtection/GooglePhotosAPIPhotosLibraryAPISearch.png "Photos Library API Search")
-
-![GooglePhotosAPIClickEnable](/images/SCALE/DataProtection/GooglePhotosAPIClickEnable.png "Click Enable")
+{{< trueimage src="/images/SCALE/DataProtection/GooglePhotosAPIClickEnable.png" alt="Enable Photos Library API" id="Enable Photos Library API" >}}
 
 ### Configure Authentication
+
 Next, click **OAuth consent screen** on the left menu, select **EXTERNAL**, then click **CREATE**.
```

Comment:
The content was still reading like it did as one long conversational procedure but now that you have added the section titles to create the summary procedure above the sections just need a little adjustment.
In the sections that follows you start by jumping into the action for that "step" like starting with "Click" or "download" so you can go that way for parallel construction:
```suggestion
Click **OAuth consent screen** on the left menu, select **EXTERNAL**, then click **CREATE**.
```